core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -162,6 +165,19 @@ public class TableTestBase {
 
   static final FileIO FILE_IO = new TestTables.LocalFileIO();
 
+  static final Map<String, String> CODEC_METADATA_MAPPING =
+      ImmutableMap.<String, String>builder()
+          .put("uncompressed", "null")
```

Comment:
nit: as mentioned on https://github.com/apache/iceberg/pull/5893 I still think this should use the enums from `AvroCodec` as it's not clear that those are actually avro codecs. I'm curious what others think here

Reply:
The `Codec` enum in `Avro` was private, so we couldn't use it. I made it public in order to use it as you suggest. In the process, I decided to rename the Map more appropriately and provided an explanatory comment. The keys are `Avro.Codec` values, while the values are now named constants from `org.apache.avro.file.DataFileConstants`.